extensions/Worker.Extensions.Abstractions/src/ExtensionInformationAttribute.cs

```diff
@@ -14,16 +14,24 @@ public class ExtensionInformationAttribute : Attribute
 
         public bool EnableImplicitRegistration { get; }
 
+        public bool SupportsDeferredBinding { get; }
```

Comment:
@jviau FYI: I think you commented here about this being `public bool SupportsDeferredBinding { get; init; }` but in the blob converter PR, the converter PR wasn't meant to have these base changes so that's my bad on PR setup, but should be fixed now. Just letting you know in case you wanted to do your review here too!

Reply:
@surgupta-msft do you want to see if `public bool SupportsDeferredBinding { get; init; }` works, if so then you can remove the new ctor

Here's the original comment for reference: https://github.com/Azure/azure-functions-dotnet-worker/pull/1108#discussion_r1010957806

Reply:
Since the existing fields of this class are also initialized in similar way, I will not block this PR, but happy to try this refactoring out separately.